### DISCUSSION TITLE
Barcode display

### DISCUSSION THREAD

**Discussion by AcernisSupport** [ant-design-blazor/ant-design-blazor#3512] [General]
Good afternoon
in a view I have to show the barcode (I use a text field for the numeric one) but I can't find the possibility to show it in bar format.
I looked in the portal but I can't find any component that does this
How can I do?
Thanks and good job

Luca Perini

**Comment by ElderJames** [comment]
Thanks for contacting us @AcernisSupport , you can use a <img> tag to show your barcode images.

**Comment by AcernisSupport** [comment]
Good evening.
Sorry for the delay but there were some problems at work.
Thanks for the reply but I don't know how to use a tag.
Do you please have a simple example just to understand how it works?
Thanks and good job

**Reply by ElderJames** [reply]
I meant `<img src="your/barcode.jpg" />`, and the jpg need generated by your back end service.

**Reply by AcernisSupport** [reply]
Good afternoon Elderjames
Thanks for the reply
I have to see how to implement it.
Good work
